_includes/releases/release-downloads-docker-image.md

```diff
@@ -70,6 +70,14 @@ This release was withdrawn, and we've removed the links to the downloads and Doc
 
 <h3 id="{{ release.version | downcase | replace: ".", "-" }}-docker-image">Docker image</h3>
 
+{% if release.docker_arm == "true" %}
+<p><a href="https://docs.docker.com/build/building/multi-platform/">Multi-platform images</a> include support for both Intel and ARM. CockroachDB on ARM systems is <b>experimental</b> and is not yet qualified for production use. Multi-platform images do not take up additional space on your Docker host.</p>
```

Comment:
No action needed, but a thought for the future about when CockroachDB on ARM systems gets fully supported, we'll need to add another column to `releases.csv` and `release-notes.py`.

Reply:
True, perhaps an `arm_prod` field. Then eventually when this is true for all supported versions, we can remove the individual platform fields.